### DISCUSSION TITLE
State is resetting after route change

### DISCUSSION THREAD

**Discussion by matt88120** [vuejs/pinia#1270] [Help and Questions]
### Reproduction

https://github.com/Lezards-Creation/Balance-ton-offre

### Steps to reproduce the bug

On top of "/", click on increment. 
Then, go to "Tutoriels" nav on the left. State is lost. 
Click on increment. Go to dashboard nav on the left. State is lost. 

### Expected behavior

Counter should stay with previous value form Dashboard or Tutoriel

### Actual behavior

Counter is always at 0

### Additional information

You can pass any email & password.

**Comment by dreambo8563** [comment]
you can provide an online demo to show your issue
https://stackblitz.com/

**Reply by jp-gouin** [reply]
@matt88120 did you make it work ? I’m running into the same issue

**Comment by LintonAchmad** [comment]
I have this issue as well.

**Comment by sir-kain** [comment]
I have the same issue, did you find something to to solve your problem?

**Comment by 13RTK** [comment]
Same problem, when I switch the route, the user info. has been reset.
![image](https://github.com/vuejs/pinia/assets/63441399/f71bd304-9535-4d9b-a58b-b368485cc171)
